tests/core/request.test.ts

```diff
@@ -237,6 +237,30 @@ describe('Request properties', () => {
 
       await fetch('/').expect(200, 'subdomains: ')
     })
+    it('assigns req.hostname', async () => {
+      const { fetch } = InitAppAndTest(
+        (req, res) => {
+          res.send(`hostname: ${req.hostname}`)
+        },
+        '/',
+        'GET',
+        options
+      )
+
+      await fetch('/', { headers: { Host: 'foo.bar:8080' } }).expect(200, 'hostname: foo.bar')
+    })
+    it('assigns req.port', async () => {
```

Comment:
Same as above.

Reply:
Same as [this](https://github.com/tinyhttp/tinyhttp/pull/439/files#r1702860768)

Reply:
`should assign the correct port to req.port by extracting it from the Host header`